tsconfig.json

```diff
@@ -0,0 +1,72 @@
+{
+  "include": [
+    "lib"
+  ],
+  "compilerOptions": {
+    /* Visit https://aka.ms/tsconfig.json to read more about this file */
+
+    /* Basic Options */
+    // "incremental": true,                   /* Enable incremental compilation */
+    "target": "es5",                          /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019', 'ES2020', or 'ESNEXT'. */
```

Comment:
agenda targets node 10 as minimum requirement if I recall correctly,
therefore
```
		"module": "commonjs",
		"target": "es2018",
		"lib": ["es2018"],
```
should be fine

i always woudl recommend adding
		"declaration": true

Reply:
I don’t believe this is needed once we switch to tslib.